tests/test_tryon_service.py

```
@@ -0,0 +1,171 @@
+"""Tryon service tests — cache, miss path, and error surfacing."""
+from unittest.mock import patch, MagicMock
+
+import pytest
+
+from server.services.tryon import TryOnResult, generate_tryon_cached
+
+
+def _make_supabase(
+    *,
+    cache_row: dict | None = None,
+    avatar_url: str = "https://x/avatar.jpg",
```

Ruff (0.15.12)

[warning] Unused function argument: `avatar_url`

(ARG001)